run-oadp-rebase.sh

```
@@ -238,8 +282,25 @@ load_config() {
     [ -n "${EXTRA_REBASEBOT_ARGS:-}" ] && unset EXTRA_REBASEBOT_ARGS
     [ -n "${SKIP_REPO:-}" ] && unset SKIP_REPO
     
-    # Always load common function config !
-    # TBD...
+    # Load version-specific variables from the SSOT before sourcing the config
+    oadp_version=$(echo "$config" | grep -oE 'oadp-1\.[0-9]+' | head -1)
+    if [ -n "$oadp_version" ]; then
+        versions_file="versions/${oadp_version}.env"
+        if [ "$source_type" = "local" ]; then
+            if [ -f "$versions_file" ]; then
+                . "$versions_file"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)